app/styles/ember-ebau-gwr.scss

```diff
@@ -1,15 +1,20 @@
 @import "ember-uikit";
 
 .ebau-gwr {
+  position: relative;
+
   #ember-ebau-gwr-modal-container {
     position: absolute;
-    width: 100%;
-    height: 100%;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
 
     div.uk-modal {
       position: relative;
-      width: 100%;
-      height: 100%;
+      padding: 100px;
+      min-height: calc(100% - 200px);
```

Comment:
Where does the fixed `200px` value come from?

Reply:
From double the padding, is there a better way?

Reply:
Hmm but the padding is based on the host app so we don't know the fixed value.

Reply:
> Hmm but the padding is based on the host app so we don't know the fixed value.

Never mind it  was already a bit late sorry :sweat_smile: 
Why do you need the padding? Is the modal overflowing the host content otherwise?

Reply:
> Hmm but the padding is based on the host app so we don't know the fixed value.

Sorry it was already a bit late :sweat_smile: Ignore that one.
Im a bit confused with the padding though. What is the reason for this fixed value / the padding in general?

Reply:
It's the padding around the modal dialog, I have to subtract the padding from the width and height of the surrounding modal background, otherwise it expands past the border of its container. If I leave the default padding, I don't know how much to subtract. I can illustrate in a few minutes.

Reply:
![image](https://user-images.githubusercontent.com/32454507/123750771-44cf8e80-d8b7-11eb-8010-947f63472d78.png)

Reply:
Okay, and cant we just  remove the padding? I still dont really get what the point of the padding is in the first place.

Reply:
We can remove it, but then the modal dialog borders the container.
![image](https://user-images.githubusercontent.com/32454507/123792256-b839c600-d8e0-11eb-8557-26b7f3841399.png)
![image](https://user-images.githubusercontent.com/32454507/123792836-5cbc0800-d8e1-11eb-8e55-9811e737631c.png)

Reply:
Aaaah okay. What you could do is check which variable this is in uikit. This way its a bit more update / integration friendly.
https://github.com/uikit/uikit/blob/bc6dd1851652e5b77387a1efefc16cea6e3d165b/src/scss/components/modal.scss
https://github.com/uikit/uikit/blob/develop/src/scss/variables.scss

Reply:
But if no easy way is possible your implementation is also very fine.

Reply:
Awesome, I was looking for something like this! I will give it a try.

Reply:
It worked, but I had to also include the media queries.